app/views/govuk_publishing_components/components/docs/cards.yml

```diff
@@ -68,6 +68,58 @@ examples:
             text: Housing and local services
             path: http://www.gov.uk
           description: Owning or renting and council services
+  one_column_layout_without_heading:
+    description: No border at the top of the list of cards is present when using a one column layout without a heading. 
```

Comment:
I think it's worth noting that a two column layout with a heading does have borders at the top of the list items.

Reply:
Done 👍